src/main/java/ui/UI.java

```diff
@@ -0,0 +1,44 @@
+package ui;
+
+import java.util.Scanner;
+
+public class UI {
+    protected static String logo =
+            " ____            _     _ _        ____ _     ___ \n" +
+                    "/ ___| _   _ ___| |   (_) |__    / ___| |   |_ _|\n" +
+                    "\\___ \\| | | / __| |   | | '_ \\  | |   | |    | | \n" +
+                    " ___) | |_| \\__ \\ |___| | |_) | | |___| |___ | | \n" +
+                    "|____/ \\__, |___/_____|_|_.__/   \\____|_____|___|\n" +
+                    "       |___/                                     ";
+    protected static final String LINEDIVIDER = "____________________________________________________________";
+    protected Scanner myScanner;
+
+    public UI(){
+        this.myScanner = new Scanner(System.in);
+    }
+
+    public void showWelcomeMessage(){
+        showLine();
+        System.out.println("Hello from\n" + logo);
+        System.out.println("What is your name?");
+        showLine();
+    }
+
+    public void showExitMessage(){
+        System.out.println("Bye, hope to see you again soon!");
+        showLine();
+    }
+
+    public void showError(Exception e){
+        System.out.println(e);
+    }
+
+    public String readCommand(){
+        System.out.print("> ");
+        return myScanner.nextLine();
+    }
```

Comment:
I checked what "Exception in thread "main" java.util.NoSuchElementException: No line found" means. I think you need to  do something like while (myScanner.hasNextLine()) then return nextLine(). If not you'll get an error for the test cases. Maybe can refer to this [link](https://stackoverflow.com/questions/7209110/java-util-nosuchelementexception-no-line-found)

Reply:
Fixed the bug, thank you for the review!